editor/timeline_displayer.gd

```diff
@@ -39,17 +44,16 @@ func _reload() -> void:
 	root.set_text(0, timeline_name)
 	root.set_text_alignment(0, HORIZONTAL_ALIGNMENT_LEFT)
 	root.set_text(columns-1, str(_current_timeline.commands.size()))
-	root.set_editable(0, true)
+	root.set_editable(0, false)
 	if not _current_timeline.changed.is_connected(_timeline_changed):
 		_current_timeline.changed.connect(_timeline_changed)
 	# See this little trick here? Is to remove the column expand.
 	# I hate it.
 	#root.set_text(columns-1, " ")
 	
-	var commands:Array = _current_timeline.commands
-	for command_idx in commands.size():
+	for command_idx in _current_timeline.commands.size():
 		var item:TreeItem = create_item(root)
-		var command:Command = commands[command_idx] as Command
+		var command:Command = _current_timeline.commands[command_idx] as Command
```

Comment:
This is exactly the same behaviour but with an extra call to `Timeline.commands` every iteration. Is there a reason?